src/mods/common/analyzer/append/bmesh.types.mod.rst

```diff
@@ -31,7 +47,24 @@
 
       :type key: int
       :mod-option arg key: skip-refine
-      :rtype: :class:`BMVert`
+      :rtype: BMVert
+      :mod-option rtype: skip-refine
+      :option function: overload
+
+   .. method:: __getitem__(key)
+
+      :type key: slice
+      :mod-option arg key: skip-refine
+      :rtype: tuple[BMVert]
```

Comment:
All blender related classes should be class-ref like:

```suggestion
      :rtype: tuple[:class:`BMVert`]
```

This can resolve dependencies automatically for the later transformer phase.
Same for the others.

Reply:
This also copies a mistake I made in #241 that I've just fixed in #277—the return type should be a variable-length homogenous tuple, not a single-item tuple.
```suggestion
      :rtype: tuple[:class:`BMVert`, ...]
```